RecoHGCal/TICL/plugins/HGCGraph.cc

```diff
@@ -221,6 +238,16 @@ bool HGCGraphT<TILES>::areTimeCompatible(int innerIdx,
           std::abs(timeIn - timeOut) < maxDeltaTime * sqrt(timeInE * timeInE + timeOutE * timeOutE));
 }
 
+template <typename TILES>
+bool HGCGraphT<TILES>::areOverlappingOnSiblingLayers(int innerIdx,
+                                                     int outerIdx,
+                                                     const std::vector<reco::CaloCluster> &layerClusters,
+                                                     float maxRSquared) {
+  float etaDiff = layerClusters[outerIdx].eta() - layerClusters[innerIdx].eta();
+  float phiDiff = layerClusters[outerIdx].phi() - layerClusters[innerIdx].phi();
```

Comment:
will this bring wraparound issues? maybe better to use a deltaR function?

Reply:
Right! replaced with deltaR2 method